pkg/reconciler/eventlistener/eventlistener.go

```diff
@@ -120,33 +114,6 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, el *v1beta1.EventListene
 	return wrapError(serviceReconcileError, deploymentReconcileError)
 }
 
-// FinalizeKind cleans up associated logging config maps when an EventListener is deleted
-func (r *Reconciler) FinalizeKind(ctx context.Context, el *v1beta1.EventListener) pkgreconciler.Event {
```

Comment:
it is ok to drop this? no more copying the config maps?

Reply:
Yeah, with this change we no longer copy around configmaps, so the finalizer to clean them up on-last-deleted is no longer necessary.